src/backup_domain.rs

```diff
@@ -22,3 +22,61 @@ use crate::pac::BKP;
 pub struct BackupDomain {
     pub(crate) _regs: BKP,
 }
+
+macro_rules! write_drx {
+    ($self:ident, $drx:ident, $idx:expr, $new:expr) => {{
+        unsafe {
+            $self._regs.$drx[$idx].write(|w| w.bits($new.into()));
```

Comment:
`$self._regs.$drx[$idx].write(|w| w.d().bits($new));`
And and `unsafe` is not needed.

Reply:
Apparently, I still need `unsafe`. I've added the `.d()` part.